### PR TITLE
Quality

### PR DESCRIPTION
**Pull request by dullus**

### CODE REVIEW

File: src/components/quality/Quality.module.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.focusFactor {
```
Comment (lukasorgovan) [inline src/components/quality/Quality.module.css:1]: what?

File: src/components/quality/Quality.module.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  font-family: 'Roboto_Regular', Arial, Helvetica, sans-serif;
+}
+
+.focusFactor h2 {
```
Comment (lukasorgovan) [inline src/components/quality/Quality.module.css:5]: what?

File: src/components/quality/Quality.module.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  font-size: 24px;
+  line-height: 26px;
+  justify-content: space-between;
+  padding: 10px;
+}
+
+.focusFactor_graphs {
```
Comment (lukasorgovan) [inline src/components/quality/Quality.module.css:12]: what?

File: src/components/quality/Quality.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+// import style from './Quality.module.css';
+
+export interface IProps {
+  sprintQualities?: I.Jira['sprintQualities'];
+}
+
+export class Quality extends React.Component<IProps> {
+  public render() {
+    if (!this.props.sprintQualities) {
+      return <div>Loading..</div>;
```
Comment (lukasorgovan) [inline src/components/quality/Quality.tsx:16]: mark todo, this is not correct, loading needs to be later added via checking the redux

Comment (lukasorgovan) [reply src/components/quality/Quality.tsx:16]: for now it should display "No data yet"

File: src/index.tsx [hunk 5]
```diff
@@ -1,23 +1,28 @@
...
+import { fetchMiddleware } from './middlewares/fetchMiddleware';
+import { mainReducer } from './reducers';
+
+import './index.css';
+
+export const HISTORY = createBrowserHistory();
 
 export const configureStore = (preloadedState?: any) => {
   const composeEnhancer: typeof compose = (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
   // eslint-disable-next-line no-shadow
```
[the remark this answers is not in the document]
Comment (dullus) [reply src/index.tsx]: done

File: src/components/quality/Quality.module.css (hunk 1 above)
Comment (dullus) [reply src/components/quality/Quality.module.css]: copypasta! done

File: src/components/quality/Quality.module.css (hunk 2 above)
Comment (dullus) [reply src/components/quality/Quality.module.css]: copypasta! done

File: src/components/quality/Quality.module.css (hunk 3 above)
Comment (dullus) [reply src/components/quality/Quality.module.css]: copypasta! done

File: src/components/quality/Quality.tsx (hunk 4 above)
Comment (dullus) [reply src/components/quality/Quality.tsx]: Done

**Pull request merged**